### ISSUE TITLE
Regression of SplitEditorAreaPane in 6.1.0

### ISSUE THREAD

**Issue by jonathanrocher** [enthought/pyface#392]
The `SplitEditorAreaPane` regressed somewhere between the release of 6.0.0 and 6.1.0, and opening a new pane to have tabs side by side is now breaking with the error of the form:
```
Traceback (most recent call last):
  File "/Users/jrocher/.edm/envs/**/lib/python2.7/site-packages/pyface/ui/qt4/action/action_item.py", line 198, in _qt4_on_triggered
    if len(argspec.argspecargs) == 1:
AttributeError: 'ArgSpec' object has no attribute 'argspecargs'
```
Issue seen on Python 2.7 with both pyside and pyqt, and both on Windows and OSX.

**Comment by rahulporuri** [comment]
@jonathanrocher can you test the fix in #393 ?

**Comment by jonathanrocher** [comment]
Yes, that fix solves this issue! Thanks @rahulporuri for the impressive reactivity!
